Dockerfile.static

```diff
@@ -8,12 +8,12 @@ WORKDIR /opt/warehouse/src/
 RUN set -x \
     && apt-get update \
     && apt-get install --no-install-recommends -y \
-        libjpeg-dev nasm
+        libjpeg-dev xvfb firefox-esr nasm
 
 # However, we do want to trigger a reinstall of our node.js dependencies anytime
 # our package.json changes, so we'll ensure that we're copying that into our
 # static container prior to actually installing the npm dependencies.
-COPY package.json package-lock.json .babelrc /opt/warehouse/src/
+COPY package.json package-lock.json /opt/warehouse/src/
```

Comment:
Why are we removing `.babelrc` here?

Reply:
I don't think that was intentional, though it's been a while 😅